support-models/src/main/scala/com/gu/support/workers/states/SendThankYouEmailState.scala

```diff
@@ -33,6 +33,8 @@ object SendThankYouEmailState {
       user: User,
       product: SupporterPlus,
       paymentMethod: PaymentMethod,
+      paymentSchedule: PaymentSchedule,
```

Comment:
the zuora step looks up the previewed payment schedule and passes it on to the email step so that it can confidently tell the user what they will be charged during the promo and standard periods